extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/discovery/AstroDiscoveryLocationChangedTask.java

```diff
@@ -0,0 +1,23 @@
+package org.eclipse.smarthome.binding.astro.discovery;
+
+import org.eclipse.smarthome.core.library.types.PointType;
+
+public class AstroDiscoveryLocationChangedTask implements Runnable {
```

Comment:
I wouldn't know why we would need this logic in its own public (and exported) class.